terraform/variables.tf

```
@@ -40,3 +46,8 @@ variable "filepath_manifest" {
   description = "Path to the Kubernetes manifest that defines the Online Boutique resources"
   default     = "../release/kubernetes-manifests.yaml"
 }
+
+variable "memorystore" {
+  type        = bool
+  description = "If true, Online Boutique's in-cluster Redis cache will be replaced with a Google Cloud Memorystore Redis cache"
+}
```

```suggestion
}

```
as per Ahrar's suggestion